docs/models.yml

```diff
@@ -126,18 +126,17 @@ user:
   is_present_in_meeting_ids:
     type: relation-list
     to: meeting/present_user_ids
-  meeting_id:
-    type: relation
-    to: meeting/temporary_user_ids  # Temporary users
-  guest_meeting_ids:
-    type: relation-list
-    to: meeting/guest_ids  # Guests in meetings
-  committee_as_member_ids:
-    type: relation-list
-    to: committee/member_ids
-  committee_as_manager_ids:
+  committee_ids:
     type: relation-list
-    to: committee/manager_ids
+    to: committee/user_ids
+
+  # committee specific permissions
+  committee_$_management_level:
+    type: template
+    replacement: committee_id
```

Comment:
This _probably_ has to say committee_id**s** since it must refer to an existing field on the model. I am not quite sure if the backend is able to handle relation list fields here, but we will see, I guess

Reply:
Same goes for all structured meeting fields which have to be changed to meeting_id**s**

Reply:
So it is already wrong everywhere? If so, it better be fixed in another PR to isolate this fix.

Reply:
No, it became wrong because you removed the field `meeting_id` (or renamed it to `meeting_ids`, whatever) to which all structured fields previously referred.

Reply:
I'm not quite sure about the meeting_id. Formerly it has been the signal for the temporary user. The meeting_ids is/should IMO a calculated field of meetings from group-assignments.
The meeting related fields are template fields (=structured fields?), which have their own meeting_id.

Reply:
Oh, you are right! I didn't even think about that. That is highly unfortunate. This probably means we have to restructure the way template relations are defined and work in the backend, since `user` does not have a direct relation field to `meeting` any more. My first idea would be two enhance the `replacement` field in this schema to take two possible values:
- A field of the same model, just like before. It must be ensured that the field exists on the model. We could theoretically (while we're at it) also ensure in the relation handling that all replacements given for the template field [1] must be present in the given relation. Should we do this? Does anyone know a field where this would not be the case?
- A CollectionField, like `meeting/id`. This also suffices to check if the referred model exists. If we want to implement the idea from above, we could add another field to the model definition for this case like `calculated_field` to indicate which field on this model corresponds to the given collection field. Then, we could implement the same check also in this case.

Let me know what you think about that or if you have other ideas.

[1] Reminder: _Template fields_ are the "main" fields with just a `$` which hold all _replacements_. When inserting the replacement in the template field's name, you get a _structured field_

Reply:
Why do you think the replacement field must exist in the model? If we want we can check in the backend, whether the user belongs ( = is member of a group of that meeting) if the user wants to store for instance an about_me_$ for meeting 222.

Reply:
Resolved: Use a new `replacement_collection` to point to the foreign collection the replacement ids belonging to.